pyogrio/geopandas.py

```diff
@@ -34,11 +33,11 @@ def _try_parse_datetime(ser):
     import pandas as pd  # only called when pandas is known to be installed
 
     if PANDAS_GE_22:
-        datetime_kwargs = dict(format="ISO8601")
+        datetime_kwargs = {"format": "ISO8601"}
```

Comment:
FWIW, I am not going to disagree merging this PR for this ;), but this is a change I dislike. When specifically creating keyword arguments, I like that `dict(..)` is closer in syntax to directly passing those to the function

Reply:
Yes, I remember the same remark being made in geopandas :-):
https://github.com/geopandas/geopandas/pull/2921#discussion_r1226338303

Reply:
Ah, it seems I somewhat consistent in my preference ;) 
It's also just adding one additional rule to the ignore list if we wouldn't want this change (C408, https://docs.astral.sh/ruff/rules/unnecessary-collection-call/)

Reply:
I'll defer to @jorisvandenbossche ; I don't personally have a strong opinion on this.  Given that it looks like this was accepted on the GeoPandas side more for tests, and this isn't strictly for tests here, it seems like ignoring `C408` might be the better option here.

Reply:
I don't have an opinion on the syntax being used. Just a mild opinion on the principle of using an oppiniated linter to avoid this kind of dicussions and then still have them :-), but I avoid having principles in general, so no problem at all to ignore `C408`.

I don't know what the basis of the remark in the geopandas comment is that it would only be for tests in geopandas. That is not the case: ruff runs on the entire geopandas codebase, so the PR also changed this in the rest of the code base.

Reply:
I was referring to this comment via the above GeoPandas link from @jorisvandenbossche re: tests:

> (now, I am fine here for our tests to just follow ruff, I just wouldn't like that for my personal scripts ;))

Reply:
Yes, I know, but the PR in geopandas isn't only targetting the tests.... Obviously it didn't change @jorisvandenbossche s personal scripts though ;-)